pkgs/development/libraries/pe-parse/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "pe-parse";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "trailofbits";
+    repo = "pe-parse";
+    rev = "v${version}";
+    sha256 = "1jvfjaiwddczjlx4xdhpbgwvvpycab7ix35lwp3wfy44hs6qpjqv";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  doCheck = true;
+  checkPhase =
+    let var = if stdenv.isDarwin then "DYLD_LIBRARY_PATH" else "LD_LIBRARY_PATH";
+    in ''
+      export ${var}=./pe-parser-library
+      dump-pe/dump-pe ../test/assets/example.exe
+    '';
```

Becomes a bit simpler as an installCheck:
```suggestion
  doInstallCheck = true;
  installCheckPhase = ''
    $bin/bin/dump-pe ../test/assets/example.exe
  '';
```

---

that's a great tip, thanks!